priv/repo/migrations/20211208194325_create_journal_entries.exs

```diff
@@ -0,0 +1,14 @@
+defmodule EChronicler.Repo.Migrations.CreateJournalEntries do
+  use Ecto.Migration
+
+  def change do
+    create table(:journal_entries) do
+      add :author, :string
+      add :title, :string
+      add :entry, :string
+      add :datetime, :utc_datetime
```

Comment:
Make sure to remove `datetime` if unused

Reply:
+1